build.gradle

```diff
@@ -44,7 +44,7 @@ ext{
     commonLangVersion = '3.5'
     materialDialogsVersion = '0.9.6.0'
     navTabStripVersion = '1.0.4'
-    constraintLayoutVersion = '1.0.2'
+    constraintLayoutVersion = '1.1.0-beta3'
```

Comment:
Is it good to use beta versions. They are still in testing.

Reply:
@abhinavraj23 I think the alpha version is available. You can use it.